### DISCUSSION TITLE
Error tidak bisa menemukan fungsi useShinydashboard

### DISCUSSION THREAD

**Discussion by MF-Faqih** [teamalgoritma/community#690] [Q&A] [answered]
saya ingin menggunakan useShinydashboard() pada UI shiny, kode yang saya gunakan:

```
navbarPage(
    title = "HomeScape",
    id = "inTabset",
    header = useShinydashboard()
    ......
```

saya sudah melakukan install package shinydashboard dan sudah memanggil library(shinydashboard) baik pada global maupun UI. Namun ketika di run menghasilkan error:

Warning: Error in useShinydashboard: could not find function "useShinydashboard"

Apakah karna versi shiny ayng saya gunakan tidak kompetible? versi shiny yang saya gunakna adalah 1.7.4

Terimakasih banyak

**Comment by ramdhinafinita** [comment]
Dear Pak @MF-Faqih 
Apakah sudah dicoba dengan code berikut:
```
navbarPage(
  title = "HomeScape",
  id = "inTabset",
  header = tagList(
    useShinydashboard()
  ),
```
Hal tersebut bukan dikarenakan dari library yang digunakan melainkan pengguanan fungsinya saja yang kurang pas
Untuk referensi bisa dilihat pada link berikut: https://stackoverflow.com/questions/73515125/r-shiny-navbarpage-values-not-loading-from-server
https://github.com/dreamRs/shinyWidgets/issues/145

**Reply by MF-Faqih** [reply]
![1](https://github.com/teamalgoritma/community/assets/105436454/a5461bbf-42b5-4f4d-9ef8-220687190410)

saya masih mendapatkan error yang sama, navbarPage yang saya gunakan saya letakkan didalam fluidPage(),namun sepertinya bukani ini masalahnya

**Reply by kinantytasyao** [reply, marked as answer]
Karena useShinydashboard berasal dari package shinyWidgets, apakah Pak Faqih sudah memanggil package tersebut?

**Reply by MF-Faqih** [reply]
Terimakasih, ternyata problemnya ada di package dan library lainnya yang harus dimasukkan, terimakasih banyak atas jawabannya kak Fini dan Kak Kinan
